Recipes/Recipes/All Recipes/Presenter/AllRecipesPresenter.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  File.swift
+//  Recipes
+//
+//  Created by Vermut xxx on 04.03.2024.
+//
+
+import Foundation
+/// протокол вью  всех рецептов
```

Comment:
```suggestion
/// Протокол вью  всех рецептов
```
Везде все классы структуры протоколы с большой буквы